chart/kubeapps/templates/kubeapps-frontend-config.yaml

```diff
@@ -79,6 +79,18 @@ data:
         {{- end }}
       }
 
+      location ~* /api/backend {
```

Comment:
So the current config already defines things like `/api/tillerdeploy` and `/api/chartsvc`, so I need `/api/<something>` to maintain the same namespacing, but I'd actually prefer to just use `/api` rather than `/api/backend`. 

Since this will be used by 3rd parties, prefixing with `/backend` doesn't really make sense or help. For example, this endpoint could/should just be `/api/apprepositories`. Thoughts?

Reply:
I agree that the `/backend` part is not adding anything to the endpoint definition so I am okay using `/api/apprepositories` instead.